fdbclient/StorageServerInterface.h

```diff
@@ -179,12 +179,13 @@ struct GetKeyValuesRequest : TimedRequest {
 	int limit, limitBytes;
 	Optional<UID> debugID;
 	ReplyPromise<GetKeyValuesReply> reply;
+	bool isFetchKeys;
```

Comment:
This is like super trivial nit, but I see most other request message structure typically have `reply` as last field. So for sake of consistency maybe we can do the same here? Don't think backward compatibility is an issue here.